### DISCUSSION TITLE
--The exclude response=PATH parameter is not available and cannot filter the output directory. Even adding this parameter resulted in no output results.

### DISCUSSION THREAD

**Discussion by zangcc** [maurosoria/dirsearch#1309] [Q&A]
### --The exclude response=PATH parameter is not available and cannot filter the output directory. Even adding this parameter resulted in no output results.
![图片](https://github.com/maurosoria/dirsearch/assets/64825932/20ad70b3-46fb-4e2f-99ab-356f006f0a06)

### Normal scan results without the - exclude response parameter：

![图片](https://github.com/maurosoria/dirsearch/assets/64825932/eec498bf-6b64-4dac-8391-5947d074b8da)

**Comment by shelld3v** [comment]
I think you just don't understand how `--exclude-response` works, maybe you are looking for `--exclude-subdirs`?
